components/brave_webtorrent/extension/assets/webtorrent-logotype.svg

```diff
@@ -0,0 +1,17 @@
+<svg viewBox="0 0 148 28" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
This doesn't render correctly for me: 
![Screen Shot 2019-08-07 at 12 51 02 PM](https://user-images.githubusercontent.com/121766/62653287-18e93f80-b912-11e9-96e6-4ed76b2e3edc.png)

Reply:
Good spot. Fixing now. I'll DM ya for my amateur context on storybook.